tox.ini

```diff
@@ -0,0 +1,27 @@
+[tox]
+envlist = py27,py34,py35,flake8,cov
+
+[testenv]
+commands = py.test {posargs}
+basepython =
+    py27: python2.7
+    py34: python3.4
+    py35: python3.5
+    flake8: python2.7
+    cov: python2.7
```

Comment:
Good point. The `test-results.xml` should be removed as well, since travis will ignore it. And travis has the concept of multiple environment, so if we were to keep the `test-results.xml` (which we wont) it wouldn't override it.